app/controllers/citizens/own_homes_controller.rb

```diff
@@ -2,12 +2,12 @@ module Citizens
   class OwnHomesController < BaseController
     def show
       @legal_aid_application = LegalAidApplication.find(session[:current_application_ref])
```

Comment:
This line can be removed as it has been replaced by legal_aid_application method